### PR TITLE
Cvp 2042 new prod

### PR DESCRIPTION
**Pull request by J0zi**
Hi team,
I have 2 questions:
1. do I need to put following stanza somewhere? Where exactly?
```
secretsync/target-namespace: "test-credentials" # The Namespace of your secret in the build clusters
secretsync/target-name: "my-secret"             # The Name of your secret in the build clusters
```
2. redhat-operator-ecosystem was renamed to redhat-openshift-ecosystem, my PR will create a new directory for the new oraganization `redhat-openshift-ecosystem`. Hope it will not break an existing.

### CODE REVIEW

**Comment by petr-muller** [comment]
/hold

Please clean up your PR: give it a sensible description, include a sensible amount of commits (likely one) with sensible commit message. Please treat this repo like you would treat any open-source repository you contribute to.

File: ci-operator/config/redhat-openshift-ecosystem/community-operators-prod/redhat-openshift-ecosystem-community-operators-prod-main__4.6.yaml
Comment (petr-muller) [inline ci-operator/config/redhat-openshift-ecosystem/community-operators-prod/redhat-openshift-ecosystem-community-operators-prod-main__4.6.yaml:34]: This is no longer necessary, ci-operator provides a writable home to tests.

File: ci-operator/config/redhat-openshift-ecosystem/community-operators-prod/redhat-openshift-ecosystem-community-operators-prod-main__4.6.yaml
Comment (petr-muller) [inline ci-operator/config/redhat-openshift-ecosystem/community-operators-prod/redhat-openshift-ecosystem-community-operators-prod-main__4.6.yaml:30]: It would be really really nice if you took this PR as an opportunity to change this very poor name for a secret. When you log in to https://vault.ci.openshift.org/ui/ and find the `pipeline-secrets` collection (you should be able to access it, Vault was populated from the previous secret mirroring configuration), there's a `tst` secret there, and it has a `secretsync/target-name` key that's currenty set to `tst`. It would be really great if you changed that to something more indicative of what's the secret's purpose.

Comment (J0zi) [reply ci-operator/config/redhat-openshift-ecosystem/community-operators-prod/redhat-openshift-ecosystem-community-operators-prod-main__4.6.yaml]: fixed in this PR, Vault will be fixed after merge to minimize downtime.

**Review by droslean — APPROVED** [review]

**Pull request merged**
